src/app/components/header/header.component.ts

```diff
@@ -15,8 +15,18 @@ export class HeaderComponent {
     interviewer: 'interviewer',
     mentor: 'mentor',
   };
+  showIt = false;
```

Comment:
Add modificator (private/public) and change name to isDisplayed, isVisible or something like that

Reply:
done